pkg/backup/item_backupper_test.go

```diff
@@ -123,6 +123,35 @@ func TestBackupItemSkipsClusterScopedResourceWhenIncludeClusterResourcesFalse(t
 	assert.NoError(t, err)
 }
 
+func TestBackupItemWhenInTerminatingStatus(t *testing.T) {
```

Comment:
Please put this in `TestBackupItemSkips`

Reply:
Done.